app/templates/events/view/tickets/access.hbs

```diff
@@ -0,0 +1,46 @@
+{{#if device.isMobile }}
+  <br><br>
+{{/if}}
+<div class="ui stackable grid">
+  <div class="row">
+    <h1 class="ui header eight wide column left floated {{if device.isMobile 'center' 'left'}} aligned">{{t 'Access Codes'}}</h1>
+    <div class="eight wide column {{if device.isMobile 'center' 'right'}} aligned"><button class="ui button primary">{{t 'Create an access code'}}</button></div>
+  </div>
+  <div class="row">
+    <div class="{{if device.isMobile 'eight wide column center' 'left'}} aligned">
+      <div class="ui very basic buttons">
+        <button class="ui button active">{{t 'All'}}</button>
+        <button class="ui button">{{t 'Active'}}</button>
+        <button class="ui button">{{t 'Inactive'}}</button>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <table class="ui very basic table">
+      <thead>
+        <tr>
+          <th>{{t '#'}}</th>
+          <th>{{t 'Access Code'}}</th>
+          <th>{{t 'Access Code URL'}}</th>
+          <th>{{t 'Accessible Ticket'}}</th>
+          <th>{{t 'Validity'}}</th>
+          <th>{{t 'Status'}}</th>
+          <th>{{t 'Action'}}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {{#each model as |entry|}}
+          <tr>
+            <td>{{entry.number}}</td>
+            <td>{{entry.access_code}}</td>
+            <td>{{entry.access_code_url}}</td>
+            <td>{{entry.accessible_ticket}}</td>
+            <td>{{entry.validity}}</td>
+            <td>{{entry.status}}</td>
```

Comment:
Status should be formatted as a label with proper colors. Again, see the current UI